From ae12e1158b6a27d94070f95f36879ce2f0da604c Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 7 Feb 2017 14:56:09 +0100
Subject: [PATCH 10/11] HMP: Fix user manual typo of
 __com.redhat_qxl_screendump

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1486479370-24026-2-git-send-email-armbru@redhat.com>
Patchwork-id: 73590
O-Subject: [RHEL-7.4 qemu-kvm PATCH 1/2] HMP: Fix user manual typo of __com.redhat_qxl_screendump
Bugzilla: 1419898
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>

Fat-fingered in RHEL-6.2 commit 1c6074d, forward ported to RHEL-7.0 in
commit faf00a8.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index dd528d2..a8ba626 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -264,8 +264,8 @@ ETEXI
     },
 
 STEXI
-@item __com.redhat_screendump @var{id} @var{filename}
-@findex __com.redhat_screendump
+@item __com.redhat_qxl_screendump @var{id} @var{filename}
+@findex __com.redhat_qxl_screendump
 Save screen from qxl device @var{id} into PPM image @var{filename}.
 ETEXI
 
-- 
1.8.3.1

